src/Sparrow.Server/Platform/Pal.cs

```diff
@@ -34,7 +34,18 @@ static Pal()
             }
             else if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
             {
-                fromFilename = Environment.Is64BitProcess ? $"{toFilename}.win.x64.dll" : $"{toFilename}.win.x86.dll";
+                var win7 = "";
+                var osdesc = RuntimeInformation.OSDescription.Split(new[] { "Windows " }, StringSplitOptions.RemoveEmptyEntries);
+                if (osdesc.Length > 1)
```

Comment:
use `PlatformDetails.IsWindows8OrNewer` for this.

Reply:
We said do not want to use that

Reply:
Why we do not want to use that?

Reply:
We decided not to use PlatformDetails (and eventually throw it away).
I can copy the methods to Pal.cs though

Reply:
Let's use them and when we will be throwing them away, you can refactor.

Reply:
done